packages/lua-language-server/unittest.patch

```diff
@@ -0,0 +1,34 @@
+--- lua-language-server/make.lua	2021-08-09 14:36:08.073522963 +0530
++++ lua-language-server-patch/make.lua	2021-08-10 01:01:28.813996570 +0530
+@@ -22,17 +22,17 @@
+         'bootstrap',
+     }
+ }
+-
+-local fs = require 'bee.filesystem'
+-local pf = require 'bee.platform'
+-local exe = pf.OS == 'Windows' and ".exe" or ""
+-lm:build 'unittest' {
+-    fs.path 'bin' / pf.OS / ('lua-language-server' .. exe), 'test.lua', '-E',
+-    pool = "console",
+-    deps = {
+-        'install',
+-        'test',
+-    }
+-}
+-
+-lm:default 'unittest'
++--
```

Comment:
Please remove the code instead of commenting it, removing it makes the patch smaller